packages/contracts/foundry.toml

```diff
@@ -1,6 +1,6 @@
 [profile.default]
 src = 'src'
-out = 'out'
+out = 'artifacts'
```

Comment:
Here I think we should actually remove all these settings and lave them as defaults. I did this on our main contracts repo and it's just a bit cleaner I think (we can't simply remove stuff, we will need to rename a couple folders here and there to make it work).